parser/parser/cc/lexer.rl

```diff
@@ -179,7 +179,7 @@ int lexer::compare_indent_level(token_t left, token_t right) {
     const auto rightLineStart = right->lineStart();
 
     // optimization: tokens start on same line
-    if (leftStart == rightStart) {
+    if (leftLineStart == rightLineStart) {
```

Comment:
Is the thought that this was just never true before, and so the optimization just never fired, but that was fine because it was also not load bearing?

Reply:
That was my expectation, yes.